backend/sales/views.py

```diff
@@ -18,10 +18,12 @@
 from buybacks.models import BuybackOrder
 from utils.permissions import CustomBasePermission
 from .parsers import XMLParser
+from .sales_record_permissions import SalesRecordsWhitelistPermission, BodySizePermission
 
 
 class ListCreateSalesReconciliationAPIView(ListCreateAPIView):
-    # permission_classes = [CustomBasePermission]
+    permission_classes = [SalesRecordsWhitelistPermission, BodySizePermission]
```

Comment:
Listing the SalesReconciliation should be authenticated (only logged in users should be able to list the SRs) so I think we need to separate List view and Create view.